provisions/roles/jenkins/slave/templates/template.json.j2

```diff
@@ -86,7 +86,8 @@
     },
     {
       "name": "BUILD_CONTEXT",
-      "description": "The build context for the container build"
+      "description": "The build context for the container build",
+      "value": "."
```

Comment:
should this be `./` to match with the defaults we have used across for this field ?

Reply:
I used `"."` based on [this](https://github.com/CentOS/container-pipeline-service/blob/master/jenkinsbuilder/cccp_index_reader.py#L113). If this commit helps make CI green, we can take @mohammedzee1000's opinion about what to set it as.

I don't think this commit should really help clear the CI, however. How is it that `master` branch is working just fine? 😞

Reply:
and I was talking about this https://github.com/CentOS/container-index/pull/234/files

Also the CI is green now. :-D

Reply:
Lets go ahead with it then :)